src/main.ts

```diff
@@ -44,6 +44,13 @@ export async function run() {
     let goVersion = (cp.execSync(`${goPath} version`) || '').toString();
     core.info(goVersion);
 
+    // get the installed version as an Action output
+    // based on go/src/cmd/go/internal/version/version.go:
+    // fmt.Printf("go version %s %s/%s\n", runtime.Version(), runtime.GOOS, runtime.GOARCH)
+    // expecting go<version> for runtime.Version()
+    let goVersionOutput = [...goVersion.split(' ')[2]].slice(2).join('');
```

Comment:
@mcdonnnj is this reliable cross-platform? Probably?

Also this is a simpler way?

```suggestion
    let goVersionOutput = goVersion.split(' ')[2].slice(2);
```

Reply:
Could probably avoid the use of `goVersionOutput` entirely too?

Reply:
@magnetikonline Sorry I had to knock the cobwebs out because it's been a long while since I looked at this pull request. I'm not sure why I felt I needed to expand it to an array and `join()` but yes your suggestion works fine. I'd also be happy to remove the variable entirely if that is preferred.

I just double-checked that this worked cross-platform in [this](https://github.com/mcdonnnj/setup-go/actions/runs/2106640282) Action run with jobs using `ubuntu-latest`, `macos-latest`, and `windows-latest`.

Reply:
Lovely 👍

Reply:
+1 to @magnetikonline's suggestion, but we can also remove the magic number '2':

```suggestion
    let goVersionOutput = goVersion.split(' ')[2].slice('go'.length);
```

Reply:
@magnetikonline @brcrista Please see [aa36c7f](https://github.com/actions/setup-go/pull/85/commits/aa36c7f7e21704beee4361666331ef3cdf913ffb) for your consideration. Please let me know if you'd like me to make any changes.

Reply:
Sorry was out for the weekend. @brcrista nice solution to the magic number `2` 👍